packages/@react-spectrum/list/test/ListView.test.js

```diff
@@ -464,7 +464,8 @@ describe('ListView', function () {
       });
     });
 
-    describe('PageUp', function () {
+    // TODO: skip since we are rendering all rows now
+    describe.skip('PageUp', function () {
```

Comment:
Many of the test skips are illustrating what kind of test will break if the virtualizer doesn't get height/width mocks (aka interactions that rely on coordinates/accurate calculated layout info will break)